### ISSUE TITLE
Suggest/request a moderator

### ISSUE THREAD

**Issue by JustinDrake** [OpenBazaar/OpenBazaar-Client#428]
As a buyer, I may not be satisfied with the moderators the buyer has selected (if any, see screenshot below). Shouldn't the buyer be able to suggest a moderator to the seller?

![screen shot 2015-12-22 at 09 45 14](https://cloud.githubusercontent.com/assets/731743/11952548/0f2342b0-a891-11e5-8ae8-dffa95c0be57.png)

**Comment by drwasho** [comment]
Yes definitely, although this is something we'll have to work towards post the initial 1.0 release
